frontend/kesaseteli/youth/public/locales/en/common.json

```diff
@@ -70,40 +70,40 @@
-      "recheck" : "ENG Tarkistathan, että olet kirjoittanut nimesi ja henkilötunnuksesi kuten ne ovat Kela-kortissasi.",
-      "validation": "ENG Tarkista seuraavat kentät: {{ fields}}"
+      "label" : "Oops! Some information is not correct.",
+      "recheck" : "Check, that you have written your name and personal number as it is on your KELA-card.",
+      "validation": "Check following form fields: {{ fields}}"
     }
   },
   "additionalInfo": {
-    "title": "ENG Tarkenna hakemustasi",
+    "title": "Please, check your application",
     "notification" : {
-      "confirmed": "ENG Sähköpostiosoite on vahvistettu",
-      "confirmedDescription": "ENG Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
-      "sent" : "ENG Kiitos! Tarkennus lähetetty!",
-      "notFound": "ENG Hakemusta ei löytynyt"
+      "confirmed": " Your e-mail address is confirmed",
+      "confirmedDescription": "",
+      "sent" : "Thanks! Your specified information has been sent!",
+      "notFound": "Your application is not found"
     },
-    "paragraph_1": "ENG Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
-    "paragraph_2": "ENG Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+    "paragraph_1": "Your application needs more information. Choose from the options below and write your explanation.",
+    "paragraph_2": "",
     "form" : {
-      "reasons": "ENG Otsikko",
-      "reasonsPlaceholder": "ENG Valitse seuraavista vaihtoehdoista",
-      "additional_info_description": "ENG Lisätiedot",
+      "reasons": "Title",
+      "reasonsPlaceholder": "Choose from following options",
+      "additional_info_description": "More information",
       "additional_info_attachments": "ENG Liitetiedostot",
       "additional_info_attachmentsHelper": "ENG Voit lisätä korkeintaan 5 liitetiedostoa. Sallitut tiedostomuodot: pdf, jpg ja png. Maksimikoko 10MB",
-      "sendButton": "Lähetä tiedot"
+      "sendButton": "Send the information"
     },
     "reasons": {
-      "unlisted_school": "ENG Kouluani ei löytynyt listalta",
-      "student_in_helsinki_but_not_resident": "ENG En asu Helsingissä, mutta käyn Helsinkiläistä koulua",
-      "moving_to_helsinki": "ENG En asu vielä Helsingissä, mutta olen muuttamassa Helsinkiin",
-      "underage_or_overage": "ENG En kuulu iän puolesta kohderyhmään, mutta vuosiluokan puolesta kyllä",
-      "personal_info_differs_from_vtj": "ENG Henkilötietoni poikkeavat väestötietojärjestelmän tiedoista",
-      "other": "ENG Muu, kerro alla"
+      "unlisted_school": "My school is not on the list",
+      "student_in_helsinki_but_not_resident": "I don't live Helsinki, but my school is in Helsink",
+      "moving_to_helsinki": "I don´t yet live in Helsinki, but I am moving there before 1 of June",
```

Comment:
Did you want to change these `don´t` into `don't`?

Reply:
True! Fixing it now